.packit.yaml

```
@@ -44,10 +44,17 @@ jobs:
         # Even though the distro context is 8.6 we are setting the SYSTEM_RELEASE_ENV to 8.7
         # to tag the updated system correctly
       distros: [centos-8.4, centos-8-latest, oraclelinux-8.6]
-  use_internal_tf: True
   trigger: pull_request
   identifier: "tier0"
   tmt_plan: "tier0"
+  # Run on Red Testing Farm Hat Ranch, tag resources to sst_conversions
+  use_internal_tf: True
+  tf_extra_params:
+    environments:
+      - settings:
+          provisioning:
+            tags:
+              BusinessUnit: sst_conversions
```

```suggestion
              BusinessUnit: rhel-sst-conversions
```
@thrix is there any convention for the tags from your POV, or should we rather align this with the rover groups?

---

Disscussed this with the team.
@bocekm said:
_> `sst_conversions` is a label that RHEL BU uses in Jira, for example. From what I know there are just RHEL subsystems - I'm not aware of seeing the term subsystem outside of RHEL. So, I'd say `sst_conversions` is fine._

So we can stick with the `sst_conversions`

---

cool, ty!